src/TLuaEngine.cpp

```diff
@@ -606,24 +605,101 @@ std::pair<sol::table, std::string> TLuaEngine::StateThreadData::Lua_GetPositionR
     }
 }
 
-sol::table TLuaEngine::StateThreadData::Lua_HttpCreateConnection(const std::string& host, uint16_t port) {
-    auto table = mStateView.create_table();
-    constexpr const char* InternalClient = "__InternalClient";
-    table["host"] = host;
-    table["port"] = port;
-    auto client = std::make_shared<httplib::Client>(host, port);
-    table[InternalClient] = client;
-    table.set_function("Get", [&InternalClient](const sol::table& table, const std::string& path, const sol::table& headers) {
-        httplib::Headers GetHeaders;
+httplib::Headers table_to_headers(const sol::table& headers) {
```

Comment:
```
/home/lion/src/BeamMP-Server/src/TLuaEngine.cpp: At global scope:
/home/lion/src/BeamMP-Server/src/TLuaEngine.cpp:608:18: error: no previous declaration for ‘httplib::Headers table_to_headers(const sol::table&)’ [-Werror=missing-declarations]
  608 | httplib::Headers table_to_headers(const sol::table& headers) {
      |                  ^~~~~~~~~~~~~~~~
```

Reply:
This is due to the new build system - either mark it `static`, or make a declaration in a header.

Reply:
Ok, I've made that function static and removed `Http.CreateConnection`